Sources/AvatarView.swift

```diff
@@ -25,33 +25,72 @@
 import Foundation
 
 open class AvatarView: UIView {
-
+    
     // MARK: - Properties
-
+    
+    public var adjustsFontSizeToFitWidth = true
     open var avatar: Avatar = Avatar()
     open var imageView = UIImageView()
+    private var radius: CGFloat?
+    private var font: UIFont? {
+        willSet {
+            if newValue != nil {
+                print("Initialized Font Size: \(String(describing: newValue?.pointSize))")
+                initialFontSize = (newValue?.pointSize)!
+            }
+        }
+    }
+    
+    open var minimumScaleFactor: CGFloat = 1.0
+    private var initialFontSize: CGFloat? {
+        willSet {
+            if newValue != nil {
+                minimumFontSize = newValue! * minimumScaleFactor
```

Comment:
Would you mind explaining the reasoning behind this. I don't understand where this is used or why it is?